File: src/main/java/co/uk/jiveelection/campaign/translator/JiveTranslator.java
package co.uk.jiveelection.campaign.translator;

import co.uk.jiveelection.campaign.output.twitter.TranslationEntity;

import java.util.List;

/**
 * Translator into Jive.
 * <p>
 * Implementations must provide their own Jive mappings.
 */
public interface JiveTranslator {
    void translate(List<TranslationEntity> entities);
}

File: src/main/kotlin/co/uk/jiveelection/campaign/translator/JiveTranslator.kt
package co.uk.jiveelection.campaign.translator

import co.uk.jiveelection.campaign.output.twitter.TranslationEntity

/**
 * Translator into Jive.
 *
 *
 * Implementations must provide their own Jive mappings.
 */
interface JiveTranslator {
    fun translate(entities: List<TranslationEntity>)
}
